apollo/src/commonTest/kotlin/io/iohk/atala/prism/apollo/base64/Base64Tests.kt

```diff
@@ -191,4 +192,78 @@ class Base64Tests {
     fun testEncodeBase16RFC_4648_6() {
         assertEquals("Zm9vYmFy", "foobar".base64PadEncoded)
     }
+
+    @Test
+    fun testStringBase64Encoded() {
+        assertEquals("aGVsbG8", "hello".base64Encoded)
+    }
+
+    @Test
+    fun testStringBase64Decoded() {
+        assertEquals("hello", "aGVsbG8".base64Decoded)
+    }
```

Comment:
We need to be much more specific with this tests, since this is a very common logic. When testing something like this, we need to search for the RFC and any test vectors or example cases, for example like the one below and the test name itself needs to reference the RFC and section.

https://datatracker.ietf.org/doc/html/rfc4648#page-12

If this doesn't exist then it should in the best case target a well know library test cases and test with their examples ;) this is as well for the tests written below.

Reply:
https://datatracker.ietf.org/doc/html/rfc4648#page-12 => We already have these tests implemented before.
These Base64 tests are related to the String and ByteArray extensions usage.